plugins/source/azuredevops/client/client.go

```diff
@@ -2,24 +2,45 @@ package client
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/microsoft/azure-devops-go-api/azuredevops"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 )
 
-type AzureDevOpsClient struct {
+type Client struct {
+	logger     zerolog.Logger
+	Connection *azuredevops.Connection
```

Comment:
just embed

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/5611/commits/27323ab43af3261f7d522153912ef810eaf8fce5